server/src/ems/ems_compositor.cpp

```diff
@@ -36,10 +36,17 @@
 #include <stdio.h>
 #include <stdarg.h>
 
-#define APP_VIEW_W (1920 / 2)
-#define APP_VIEW_H (1080)
+// native quest resolution
+// #define APP_VIEW_W (1832)
+// #define APP_VIEW_H (1920)
 
-#define READBACK_DIV_FACTOR (4)
+
+#define APP_VIEW_W (1920)
```

Comment:
Is there any gain is pushing this value above 1832 ? since native Quest2 res is 1832x1920, wouldn't it be cleaner/sufficient to set APP_VIEW_W to 1832 here ?

Reply:
I was just monkeying with the values that were already there. It gets divided below, etc.

If the server builds and runs with the simpler values you give that's fine, but IIRC I couldn't get it much over 920?

Reply:
yeah, 1832/2 = 916, whereas 1920/2=960, which is over 920, hence my confusion.
If you're positive about this change, as in.. it's been tested working, then I'll go for it.

Reply:
The change as it stands is tested working. It's certainly something to clean up in the future though.

Reply:
Happy with this, merging.